### ISSUE TITLE
Some kind of state on whether a subscriber has received a specific email or not

### ISSUE THREAD

**Issue by alexknowshtml** [brennandunn/automationspec#1]
One thing that I have come to like about ConvertKit's sequences is that an email in a given sequence won't be sent twice (and that it'll be automatically skipped over). 

Except sometimes I want to override it. Especially in launches. 

It'd be good to maintain some sort of state of which subscribers have gotten which emails, maybe even a count on how many times, and if I'm being SUPER ambitious to track if the markdown file changed at all between sends (deferring actual change tracking to a diff, but at least tell me if revisions exist)

**Comment by brennandunn** [comment]
Maybe...

```ruby
send_email :launch_email, resend: true
```

Or defined at the flow level:

```ruby
class BasicAutomation < Flow
  resend_emails
```

**Comment by alexknowshtml** [comment]
yeah something like that. would this be the right place to set a flag to count/diff too, or I guess it could do that by default and be surfaced elsewhere.
